openvpn-monitor.py

```diff
@@ -25,6 +25,7 @@
 import socket
 import re
 import argparse
+import geoip2.database
```

Comment:
Will this give an ImportError if only geopv1 is installed?

Reply:
Yes, you could solve with:
try:
    import geoip2.database
except:
    # No geoip2 support
    pass

Reply:
Could you fallback to geoip1 support in that case?